### PR TITLE
Test File for Button Component

### PR DESCRIPTION
**Pull request by Killer2OP**
## Title
Test file

## Check List (Check all the applicable boxes)

- [ ] Bug(s) fixed
- [ ] Enhancement done
- [X] New Feature(s) added
- [ ] Documentation changed
- [ ] All new and existing tests passed.
- [ ] My code follows the code style of this project.

## Screenshots (if any)

image

### CODE REVIEW

**Comment by Killer2OP** [comment]
wait for me to include all the files then merge it.

**Comment by Killer2OP** [comment]
@thekavikumar Now you can review the file.

**Comment by Killer2OP** [comment]
@thekavikumar Can you please review the PR

File: package.json [hunk 1]
```diff
@@ -1,9 +1,8 @@
 {
   "name": "love-simple-ui",
-  "version": "1.0.26",
+  "version": "1.0.24",
```
Comment (thekavikumar) [inline package.json:3]: pls update the crt version

File: package.json [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
   "description": "An open-source UI library for developers to make their life easier :)",
   "main": "dist/index.js",
-  "module": "dist/index.mjs",
```
Comment (thekavikumar) [inline package.json:6]: why did you remove it?

File: package.json [hunk 3]
```diff
@@ -45,29 +43,73 @@
...
     "@ladle/react": "^2.12.3",
     "@size-limit/preset-big-lib": "^8.2.6",
-    "@types/react": "^18.2.14",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^14.0.0",
+    "@types/jest": "^29.5.2",
+    "@types/react": "^18.2.1",
     "@types/react-dom": "^18.2.2",
     "autoprefixer": "^10.4.14",
-    "clsx": "^1.2.1",
```
Comment (thekavikumar) [inline package.json:53]: add the deps

File: package.json [hunk 4]
```diff
@@ -45,29 +43,73 @@
...
     "tsup": "^6.7.0",
     "typescript": "^5.0.4"
   },
   "dependencies": {
-    "goober": "^2.1.13"
+    "clsx": "^1.2.1",
+    "goober": "^2.1.13",
+    "ladle": "^0.0.0",
+    "react-helmet": "^6.1.0",
+    "tailwind-merge": "^1.13.2"
```
Comment (thekavikumar) [inline package.json:77]: add these as dev deps

**Review by thekavikumar — CHANGES_REQUESTED** [review]
resolve the comments!

**Comment by Killer2OP** [comment]
> resolve the comments!

OK

**Comment by Killer2OP** [comment]
@thekavikumar DONE.

**Pull request closed without merging**